crates/analytics/src/payment_intents/accumulator.rs

```diff
@@ -117,7 +117,7 @@ impl PaymentIntentMetricAccumulator for SmartRetriedAmountAccumulator {
             .amount_without_retries
             .and_then(|i| u64::try_from(i).ok())
             .or(Some(0));
-        (with_retries, without_retries)
+        (with_retries, without_retries, Some(0), Some(0))
```

Comment:
Nit:
It would make more sense to store `Some(0)` in a variable and give that variable a name to indicate what that value is